src/Microsoft.IdentityModel.Clients.ActiveDirectory/Features/CertificateAssertion/ClientAssertionCertificate.cs

```diff
@@ -64,16 +65,33 @@ public ClientAssertionCertificate(string clientId, X509Certificate2 certificate)
 	            if (certificate.PublicKey.Key.KeySize < MinKeySizeInBits)
 #endif	
             {
-                throw new ArgumentOutOfRangeException("certificate",
+                throw new ArgumentOutOfRangeException(nameof(certificate),
                     string.Format(CultureInfo.InvariantCulture, AdalErrorMessage.CertificateKeySizeTooSmallTemplate, MinKeySizeInBits));
             }
 
             this.clientId = clientId;
             this.Certificate = certificate;
         }
 
+		/// <summary>
+		/// Constructor to create credential with client Id and certificate and claimSet.
+		/// </summary>
+		/// <param name="clientId">Identifier of the client requesting the token.</param>
+		/// <param name="certificate">The certificate used as credential.</param>
+		/// <param name="additionalClaims">The additional claims to be added to the payload of the client assertion.</param>
+		public ClientAssertionCertificate(string clientId, X509Certificate2 certificate, Dictionary<string, string> additionalClaims) : this(clientId, certificate)
```

Comment:
There seems to be some odd spacing here.

Reply:
Consider when chaining :this on separate line.
public ClientAssertionCertificate(....)
    : this(...)